db/post_migrate/20210530122334_move_existing_triggers_to_fields.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class MoveExistingTriggersToFields < ActiveRecord::Migration[6.1]
+  def create_field(automation, component, name, metadata)
+    automation.fields.create!(
+      component: component,
+      name: name,
+      metadata: metadata,
+      target: 'trigger'
+    )
+  end
+
+  def change
+    DB.query("SELECT name,automation_id,metadata FROM discourse_automation_triggers").each do |trigger|
+      automation = DiscourseAutomation::Automation.find(trigger.automation_id)
+
+      automation.update_column(:trigger, trigger.name)
+
+      trigger.metadata.each do |key, value|
+        if key == 'group_ids' && trigger.name == DiscourseAutomation::Triggerable::USER_ADDED_TO_GROUP
```

Comment:
All those `DiscourseAutomation::Triggerable::USER_ADDED_TO_GROUP` variable should be replace their value instead. Otherwise if we ever change their value in the future, this migration might break :(